examples/knowledge_quiz/example_question.html

```diff
@@ -0,0 +1,69 @@
+<h1 id="quiz-introduction">Quiz introduction</h1>
+
+<p>The quiz can have normal HTML before it, which will just be treated like normal markdown.</p>
+
+<p>Anything inside a <code>---question---</code> block will be parsed into a <code>&lt;form&gt;</code> representing the quiz question</p>
+
+<p>A question may optionally have a ‘frontmatter’ section with the <code>legend</code> parameter to override the default ‘Question’ string used in the form’s legend tag.</p>
+
+<form class="knowledge_quiz__question">
+  <fieldset>
+    <legend>Question 1 of 3</legend>
+    <div class="knowledge_quiz__blurb">
+      <p>The bug sprite starts facing upwards.</p>
+
+<p><img src="./img/q3-1.png" alt="Bug facing upwards" /></p>
```

Comment:
are we unable to get this to indent, not a big deal, but it makes me v. sad

Reply:
Could not figure how to do quickly originally - was thinking of looking at it after I'd got everything working.

Guess I'll have to do a lot of this:
```
foo_html.join('      ').strip
```

Reply:
I had a go at this - was doing okay until I hit the `<ul>` - but it leaves the code looking really messy / weird, so decided that it's better that this is legible / consistent.  If we want pretty-printed html, we should run it through nokogiri.